### PR TITLE
added timer when waiting for a decision of the player for starting / …

### PR DESCRIPTION
**Pull request by PrinzPizza42**
…ignoring the minigame. When the time is up the items get dropped to not give the player an unfair advantage

### CODE REVIEW

File: src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java [hunk 1]
```diff
@@ -1,19 +1,72 @@
...
+        deaths.remove(player.getUniqueId());
+        inventories.remove(player.getUniqueId());
+    }
+
+    private static void decidedTimer(Player player) {
+        new BukkitRunnable() {
+            public void run() {
+                if(i > 0) {
+                    if(!playerDecided) {
+                        player.sendTitle("§6Entscheide dich im Chat", "§6Du hast noch §c" + i + "§6 Sekunden zeit", 0, 25, 10);
```
Comment (JonasFranke) [inline src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java:40]: Use components

**Review by JonasFranke — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java (hunk 1 above)
Comment (PrinzPizza42) [reply src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java:40]: the problem with titles is, that they dont accept components, i tried that already but they just accept strings and if i put .toString() behind the component it doesnt work at all

File: src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java [hunk 2]
```diff
@@ -1,19 +1,72 @@
...
+                    }
+                    else {
+                        i = 10;
+                        RespawnListener.playerDecided = false;
+                        Main.getPlugin().getLogger().info("runnabel cancelled");
+                        cancel();
+                    }
+                }
+                else {
+                    player.sendTitle("", "Dein Inventar wird an deinen Todesort gedroppt", 10, 40, 10);
```
Comment (JonasFranke) [inline src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java:51]: replace with showTitle as in line 43ff

File: src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java (hunk 1 above)
Comment (JonasFranke) [reply src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java]: I pushed changes to your branch. The `sendTitle()` is deprecated as seen in the javadocs and replaced by `showTitle()`

File: src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java (hunk 2 above)
Comment (PrinzPizza42) [reply src/main/java/org/example/DeathMinigames/listeners/RespawnListener.java:59]: I will do that later

**Pull request merged**